internal/parser.go

```diff
@@ -111,13 +110,12 @@ func (p *Parser) parseTasks() error {
 		return err
 	}
 
-	re := regexp.MustCompile(osCommandRegexp)
-	allFilesPaths := []string{}
+	var allFilesPaths []string
```

Comment:
The variable initialization to an empty array is deliberate, so that I don't have to check against nil when iterating over it.

Reply:
@reneManqueros One more instance left here and this is good to go

Reply:
@dugajean done